Guide/recipes.markdown

```diff
@@ -48,6 +48,17 @@ instance CanRoute StaticController where
 
 Now the terms can be reached at `/terms` instead of `/Terms`. The about is at `/about` now, instead of `/About`.
 
+Add the static controller to the `Web.FrontController`:
```

Comment:
@eclecticlly maybe let's also mention above that we should have also `instance AutoRoute StaticController` commented out - as otherwise, I believe it takes over our defintion.

Reply:
That step is already taken into account above @amitaibu.